### PR TITLE
ex 08 complete

### PR DESCRIPTION
**Pull request by vinc86**
please check!

### CODE REVIEW

File: 02-PB-operators-arithmetic-pt1/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
+
+
+<<<<<<< HEAD
+/* TASK 0 */
```
Comment (martl) [inline 02-PB-operators-arithmetic-pt1/index.js:4]: remove this line 3

File: 02-PB-operators-arithmetic-pt1/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+// 10. my question: the answer should be 14 but why is the machine calculation result 12?
+let d = b - a;
+console.log(d);
+console.log(d + c);
+
+// 11.
+console.log(resultOne % resultTwo);
+>>>>>>> 2b144e551a9dffbf5d09cf3e123c1b016f750e5b
```
Comment (martl) [inline 02-PB-operators-arithmetic-pt1/index.js:113]: remove this line 113 too

File: PB-language-statements/index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+console.log("Hello World!");
```
Comment (martl) [inline PB-language-statements/index.js:1]: please rename the folder to 01-PB-language-statements

File: 02-PB-operators-arithmetic-pt1/index.js (hunk 1 above)
Comment (vinc86) [reply 02-PB-operators-arithmetic-pt1/index.js:1]: done!

File: 02-PB-operators-arithmetic-pt1/index.js (hunk 2 above)
Comment (vinc86) [reply 02-PB-operators-arithmetic-pt1/index.js]: fixed!

File: PB-language-statements/index.js (hunk 3 above)
Comment (vinc86) [reply PB-language-statements/index.js]: also edited!

**Pull request closed without merging**